Stratis.Bitcoin/Utilities/LinqExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.Linq
+{
+    public static class LinqExtensions
+    {
+        public static long Median(this IEnumerable<long> source)
```

Comment:
I'd like to stick with this since it has the same format as the Linq extensions for Average, Min, Max, etc